packages/core/src/Stores/ui-store.js

```diff
@@ -100,6 +103,9 @@ export default class UIStore extends BaseStore {
         error_message     : '',
     };
 
+    // account switcher tab states
+    @observable account_switcher_tab_index = 0;
```

Comment:
Do we need a state for this? Shouldn't it open the user's current login ID type on open of the account switcher?

Reply:
Yes, I discussed with UI/UX and they assume users will want switch to the tabs they want upon closing the switcher, so we should remember we `tab_index` they previously selected, initially the states were handled within and not stored in the `ui_store`.

Reply:
As per further discussion, this was changed, as the account switcher should default to the current logged in account type on open.